docs/user-guide/cli-install-cli-next.md

```diff
@@ -0,0 +1,57 @@
+# Install the Zowe CLI @next version
+
+<Badge text="Technical Preview"/> The Zowe CLI @next release is a technical preview. Technical previews are for testing only and are not ready for production. Your feedback is valued and appreciated.
+
+Install the Zowe CLI `@next` version from the online registry. You can follow this procedure for a first-time installation, or to update a currently installed version.
```

Comment:
You can now also install the `@next` version from the [Zowe Downloads](https://www.zowe.org/download.html) site under the Zowe Technical Preview section. This would be worth mentioning (and providing specific instructions) for those who cannot access the public npm registry.

Reply:
I added a section for downloading and installing from the Zowe Downloads page.